server/src/routes/githubTest.ts

```diff
@@ -0,0 +1,11 @@
+import { Request, Response } from 'express';
+const express = require('express');
+const app = express();
+
+app.get('/github', (req: Request, res: Response) => {
```

Comment:
Use the express.router module.  Look at `server/routes/routes.ts` page.  Call the path with an `axios` call in the client end